tests/gold/arrayInStruct.c

```diff
@@ -3,8 +3,8 @@
 
 void arrayInStruct(struct awl_unsignedS32 * v0, struct awl_unsignedS32 * out)
 {
-  struct s_2_unsignedS32_awl_unsignedS32 e0 = { 0 };
-  struct s_2_unsignedS32_awl_unsignedS32 v6 = { 0 };
+  struct s_2_1unsignedS32_1awl_unsignedS32 e0 = { 0 };
```

Comment:
Could the format be `s_2_Nxtype..` so we get a delimiter between the fields that indicate multiplicity?

Reply:
Sure, if you prefer that. It's a tossup to me.